### PR TITLE
Event sessions

### PR DESCRIPTION
**Pull request by ianlovell**

### CODE REVIEW

**Comment by jebbinSCL** [comment]
Where are the pictures from? Are they scott logic people?

**Comment by jebbinSCL** [comment]
:shipit:

**Comment by ianlovell** [comment]
Assumed they were public domain, but will check with UX

File: app/containers/Calendar.jsx [hunk 1]
```diff
@@ -1,35 +1,77 @@
...
+              speakerForename={session.speakerForename}
+              speakerSurname={session.speakerSurname}
+              speakerImageUri={session.speakerImageUri}
+            />
+          </div>
+        );
     }
 
     render() {
+        console.log(this.props.events);
```
Comment (lclarksl) [inline app/containers/Calendar.jsx]: This shouldn't go into production

**Comment by ianlovell** [comment]
Will swap images out, so don't merge!

File: app/components/Calendar/EventSession.jsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            <div style={styles.base.card}>
+              <div style={styles.base.speaker}>
+                <img style={styles.base.speakerImage} src={this.props.speakerImageUri} />
+                <div dangerouslySetInnerHTML={{ __html: bio }}>
+                </div>
+              </div>
+              <div style={styles.base.session}>
+                <div style={styles.base.sessionSummary}>
+                  <div style={styles.base.name}>
+                     <p>{`${this.props.speakerForename} ${this.props.speakerSurname}`}</p>
```
Comment (lclarksl) [inline app/components/Calendar/EventSession.jsx]: We'll need to check for null values on these

Comment (jebbinSCL) [inline app/components/Calendar/EventSession.jsx]: Maybe we should use options in the front end as well :stuck_out_tongue_winking_eye:

**Comment by lclarksl** [comment]
This looks sweeeeet :shipit: after comments

File: app/components/Calendar/EventSession.jsx (hunk 2 above)
Comment (ianlovell) [inline app/components/Calendar/EventSession.jsx]: Yes, will add a task to review the PropTypes for 'isRequired'

**Pull request merged**